apps/src/applab/commands.js

```diff
@@ -1381,7 +1384,9 @@ applabCommands.startWebRequest = function (opts) {
   apiValidateType(opts, 'startWebRequest', 'callback', opts.func, 'function');
   var req = new XMLHttpRequest();
   req.onreadystatechange = applabCommands.onHttpRequestEvent.bind(req, opts);
-  req.open('GET', opts.url, true);
+  var url = XHR_PROXY_PATH + '?u=' + encodeURIComponent(opts.url.replace(/ /g, '%20')) +
```

Comment:
Can you tell me more about why we need to manually replace spaces with %20 before the call to encodeUriComponent?  encodeUriComponent already encodes a space with %20.  In this code the space ends up double escaped as %25%20  (where %25 is the escaped '%').